2025-01-25 09:11:26 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /R0ozgCnf%2F889478%2F67830dacb757176f73ad4a14bac5170a5b9b7b2c%2F6794aa8075f07268d5fc4ffd80ff8ec80c4e319d?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=5FDF8CCC449C41A397FEE694FC478716&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737799824&x-oss-signature=mC8vG6mshqbujgu24PmkrYOTCw70%2FaFAk4VFAXVcIBo%3D&x-oss-signature-version=OSS2
2025-01-25 09:11:30 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2025-01-25 09:13:20 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /0xXyq1K5%2F889478%2F62f77c7e193a2a12a5e548f5a8f2746e77adea36%2F6794ab12d9a0d85440b64278b3100114105718c4?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=D078B54B6BF44C0987CEC56D1E2DAEDA&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737799970&x-oss-signature=LXG%2F%2BscguKYVlcLggMkKGrfGkvVByNypcVDTv4fXIz4%3D&x-oss-signature-version=OSS2
2025-01-25 09:13:53 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /0xXyq1K5%2F889478%2F62f77c7e193a2a12a5e548f5a8f2746e77adea36%2F6794ab12d9a0d85440b64278b3100114105718c4?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=D078B54B6BF44C0987CEC56D1E2DAEDA&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737799970&x-oss-signature=LXG%2F%2BscguKYVlcLggMkKGrfGkvVByNypcVDTv4fXIz4%3D&x-oss-signature-version=OSS2
2025-01-25 09:16:21 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /CiPjjSB8%2F889478%2F66bdd74d555a4f353f1a410293b38e575249ef13%2F6794abc6fb7e924f00804cbd84bf10107a2125bb?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=5E790B5FEA1E40ED99549A569873EAED&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800151&x-oss-signature=8lcstMlNdajf4uidNroAbsF0%2Ff8c4%2By6aifNiQVmk1U%3D&x-oss-signature-version=OSS2
2025-01-25 09:16:52 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /CiPjjSB8%2F889478%2F66bdd74d555a4f353f1a410293b38e575249ef13%2F6794abc6fb7e924f00804cbd84bf10107a2125bb?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=5E790B5FEA1E40ED99549A569873EAED&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800151&x-oss-signature=8lcstMlNdajf4uidNroAbsF0%2Ff8c4%2By6aifNiQVmk1U%3D&x-oss-signature-version=OSS2
2025-01-25 09:20:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377968272898277eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:20:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:20:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-25 09:20:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377968285458358eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:20:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:20:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-25 09:20:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377968308048556eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:20:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:20:33 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-25 09:20:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377968340688838eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:20:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:20:38 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-25 09:20:38 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/446045/Kacyu | 卡丘 - 446045.
2025-01-25 09:22:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377969272641175eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:22:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:22:08 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-25 09:22:08 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377969285201241eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:22:08 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:22:10 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-25 09:22:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377969307751387eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:22:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:22:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-25 09:22:14 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701a17377969340311644eecef","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:22:14 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:22:18 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-25 09:22:18 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/446045/Kacyu | 卡丘 - 446045.
2025-01-25 09:23:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /jB0WRtTi%2F889478%2F634f250cff7fbf3a55a0492b84d981d7b40b246b%2F6794ad81934f5872a1324815829bb2228e3bcc68?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=670F30CBDA854ACB909481A2C1575062&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800593&x-oss-signature=j8yPMMyxLhgvtz5xZaEeev0SwAakUPhG4rfwZGc5m4Q%3D&x-oss-signature-version=OSS2
2025-01-25 09:23:47 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/jB0WRtTi%2F889478%2F634f250cff7fbf3a55a0492b84d981d7b40b246b%2F6794ad81934f5872a1324815829bb2228e3bcc68?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=670F30CBDA854ACB909481A2C1575062&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800593&x-oss-signature=j8yPMMyxLhgvtz5xZaEeev0SwAakUPhG4rfwZGc5m4Q%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6794ADA254BDDE313106D61C</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E3B84C17EE4D6D5A00EFE6FF8F19ECA3"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-25 09:23:48 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/jB0WRtTi%2F889478%2F634f250cff7fbf3a55a0492b84d981d7b40b246b%2F6794ad81934f5872a1324815829bb2228e3bcc68?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=670F30CBDA854ACB909481A2C1575062&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800593&x-oss-signature=j8yPMMyxLhgvtz5xZaEeev0SwAakUPhG4rfwZGc5m4Q%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6794ADA3243EF030398E54CC</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E3B84C17EE4D6D5A00EFE6FF8F19ECA3"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-25 09:23:48 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/jB0WRtTi%2F889478%2F634f250cff7fbf3a55a0492b84d981d7b40b246b%2F6794ad81934f5872a1324815829bb2228e3bcc68?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=670F30CBDA854ACB909481A2C1575062&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800593&x-oss-signature=j8yPMMyxLhgvtz5xZaEeev0SwAakUPhG4rfwZGc5m4Q%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>6794ADA43CA53737383D0A6E</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"E3B84C17EE4D6D5A00EFE6FF8F19ECA3"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2025-01-25 09:23:48 - ERROR - aliyundrive.py - 1077 - File tmp/98347899/98347899.json.txt Part 1 upload failed.
2025-01-25 09:24:22 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /9Ca7zGec%2F889478%2F62a5be47ed106ce3e08f457db787946acc10c4eb%2F6794ada8faa4bc05a3b744e0b8aefd8f5caf8d67?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5WDeNLEirwYwaqaZGjEvEU2bdxeta3e1Tz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNWutOVz1q1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneA%2F1Hyktb7uHwufJ7FxfIREfquk63pvSlHLcLPe0Kjzzleo2k1XRPVFF%2B535IaHXuToXDnvSiwRmTGfXtuMkagAGPwoaWWbVLgu2O0z69Sgu3e5Gyoqfc6gyNDjaF0iJvA01gHjgXgHoRmM2kZ9sMTEmopP%2F8PNf4OcUlRPnUEEKTLySFFFoUt6Fa4vyVSxJzO3iCDmWvLzE7QcNbNvUD2g8RVsbeLGeL1DL7a5%2Bz4F%2FU93xnM%2BoTw7Q%2FbX39OuuWQSAA&uploadId=B3A77A9536D346FC8381A832FEEBB7C2&x-oss-access-key-id=STS.NV63hpgc9vhqfNuXEcaSrZhu7&x-oss-expires=1737800632&x-oss-signature=r7JXVN3d82OMThQ%2F0BhDcelXs7CUBje4UhszA9fI28A%3D&x-oss-signature-version=OSS2
2025-01-25 09:25:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917377971244587017ed92f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:25:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:25:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-25 09:25:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917377971257327140ed92f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:25:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:25:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-25 09:25:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917377971280077395ed92f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:25:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:25:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-25 09:25:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700917377971312887759ed92f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:25:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:25:35 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-25 09:25:35 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/446045/Kacyu | 卡丘 - 446045.
2025-01-25 09:26:28 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/11/10/00/02/50/124145522_p0.jpg
2025-01-25 09:28:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17377973145278384e8eed","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:28:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:28:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2025-01-25 09:28:35 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17377973158018572e8eed","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:28:35 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:28:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2025-01-25 09:28:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17377973180798891e8eed","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:28:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:28:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2025-01-25 09:28:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00801c17377973213681322e8eed","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2025-01-25 09:28:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2025-01-25 09:28:45 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2025-01-25 09:28:45 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/446045/Kacyu | 卡丘 - 446045.
